src/Pages/Map/List.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export default function List(props) {
+  const { searchList } = props;
```

Comment:
1. arrow function으로 함수형 컴포넌트를 작성할 수도 있습니다
2. props에서 직접 비구조화 할당을 진행할 수 있습니다.
```suggestion
export default const List = ({ searchList }) => {
```

Reply:
export default const List = ({ searchList }) => { 
종택님 이렇게하니까 빨간줄 뜨고 function expression이 아니라고 안된다고해서 저렇게했는데 .. 가르침을 주십시오..